blogs.json

```diff
@@ -7601,9 +7601,16 @@
             "author": "Aya Baghdadi",
             "site_url": "https://www.youtube.com/channel/UC7554uvArdSxL4tlws7Wf8Q",
             "feed_url": "https://www.youtube.com/feeds/videos.xml?channel_id=UC7554uvArdSxL4tlws7Wf8Q"
+          },
+          {
+            "title": "GoMarketMe Blog",
+            "author": "Toni Peinoit",
+            "site_url": "https://gomarketme.co/blog",
+            "feed_url": "https://gomarketme.co/feed",
+            "twitter_url": "https://x.com/gomarketmetech"
```

Comment:
Thanks for this, but I have a feeling you have mistakenly added this as an Arabic YouTube channel. It should be in English > Marketing and Business Blogs.

Reply:
My bad, you are right. I just pushed a fix. Thank you